K8S.md

```diff
@@ -24,12 +24,34 @@ This project deploys an AI pipeline with a multipurpose front end for text gener
 
 Get troubleshoot help on the [FAQ](FAQ.md)
 
+## Requirements
+
+You need to be and administrator.
+
+> If not should should have enough privileges for OKE, Network, and Database services. Plus others like tenancy inspeact. See example:
```

Comment:
double "should".  I believe you want "If not, you should have..."
"inspeact" should be "inspect" I believe.

Reply:
Changes implemented! Thank you @peppertech !